Help/MAC_Hints.rst

```diff
@@ -1,25 +1,74 @@
 Coverage Installation Hints for macOS Users:
 ============================================
 
+pyenv
+-----
+
+Here we use pyenv, which is simply a python installer that allows
+you to install multiple python versions and switch through them.
+In case your macOS has an older version of python that coala doesn't
+support we recommend using pyenv to install and shift to a newer
+supported version. After installation, you may also want to set up
+a virtual environment using :code:`venv` or :code:`virtualenv`.
```

Comment:
I think by default macOS has Python 2.7 installed (at least that was the case for me) so by saying "shift to newer supported version" I tried to mean that using pyenv they install python > 3.3 that is a newer and a version with which coala works. (and at the beginning I mentioned that pyenv is for installing different python version and switch through them)

now this may be helpful for people who have an older python version in their os, however there are other ways as well that users can do easily (like using homebrew)

so do I remove the pyenv part? @SanketDG @jayvdb